src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -290,7 +310,7 @@ Type[] IInvocationBinder.GetParameterTypes(string methodName)
             Type[] types;
             if (!_paramTypes.TryGetValue(methodName, out types))
             {
-                throw new InvalidOperationException($"The hub method '{methodName}' could not be resolved.");
+                return Type.EmptyTypes;
```

Comment:
Yeah, the concern I had about this was that because we allow mixing formats different connections will behave differently for the same hub method and parameters...